### ISSUE TITLE
add mapping quality cutoff in callpeak subcommand for BAM and BAMPE

### ISSUE THREAD

**Issue by carl-** [taoliu/MACS#130]
please consider add mapping quality cutoff for BAM and BAMPE format input during peak calling

thanks

**Comment by taoliu** [comment]
Good point. But why not to polish BAM or BAMPE file first using Picard or Samtools? More manipulations on the raw files are out of the scope of MACS tool.

**Comment by carl-** [comment]
filtering out of read according to their mapping quality using Picard or samtools will output many files, this will take much disk space, especially when the original BAM file is large. but by setting a cutoff during MACS running time, this will be very helpful to evaluate the impact of mapping quality for peak calling if this feature is supported in the feature release,
